doc/user/mirrors.md

```diff
@@ -110,7 +99,7 @@ export TIUP_MIRRORS=/path/to/mirror
 tiup cluster xxx
 ```
 
-`/path/to/mirror` is the location of <target-dir> in `tiup mirrors <target-dir>`, or if in /tmp/package:
+`/path/to/mirror` is the location of <target-dir> in `tiup mirror <target-dir>`, or if in /tmp/package:
```

Comment:
This should be `tiup mirror set <target-dir>` i think.

Reply:
```suggestion
`/path/to/mirror` is the location of <target-dir> in `tiup mirror set <target-dir>`, or if in /tmp/package:
```